### DISCUSSION TITLE
Popular Extension Already Done for Hyva Theme but not for Breeze Theme

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#103] [3. Integrations]
Dear Breeze Theme Team,

I am writing to inquire about the compatibility of several extensions we are currently using on our website. We believe that the Breeze Theme is a great fit for our website and would like to switch to this theme, but we need to ensure that the following extensions are compatible with it:

Amasty_ImageOptimizer
Amasty_ImageOptimizerUi
Amasty_Scroll * (infinity scroll)
Amasty_Sorting *
Amasty_Xlanding
Apptrian_FacebookPixel *
Apptrian_PinterestPixel
Apptrian_TikTokPixel
Astound_Affirm *
Faonni_Price
Genaker_Opcache
MagePal_GoogleTagManager *
MagePal_EnhancedEcommerce
MagePal_DataLayer
Magedelight_Base
Magedelight_CustomForm
Mageplaza_AutoRelated
Mageplaza_QuickCart 
Mageplaza_Smtp *
PayPal_Braintree
StripeIntegration_Payments
MagicToolbox_MagicZoomPlus
MagicToolbox_MagicScroll
Mirasvit_GeoIp
Revenuehunt_ProductQuiz
Smartwave_Filterproducts
Smartwave_Megamenu
Stamped_Core *

*neccessary ones

We understand that these extensions are necessary for the frontend functionality of our website and would like to know if they will work seamlessly with the Breeze Theme. If they are not compatible, we are open to exploring alternative extensions that offer the same functionality and are compatible with the Breeze Theme.

We would greatly appreciate any information or recommendations you can provide regarding this matter. We believe that the Breeze Theme, in combination with compatible extensions, will greatly benefit our business and improve our overall website performance.

Thank you for your attention to this request, and I look forward to hearing back from you soon.

Best regards,

**Comment by dimdimich** [comment]
Hi,

first of all i would suggest start checking list of compatible modules at https://breezefront.com/extensions. You  can also check compatible modules that offer similar features. 

If you can't find this modules there please contact extension developers and ask them to add compatibility with Breeze. We already had discussion with many extension vendors. They are open to add Breeze integration from their side in case they will have many requests from their clients.

Also please notice that backend and checkout modules will work right out of the box and doesn't require integration.
